samples/interactive-tutorials/search/utils.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import shlex
+import subprocess
+
+
+def get_project_id():
+    """Get the ProjectId from the gcloud configuration"""
+    get_project_command = "gcloud config get-value project --format json"
```

Comment:
Could this break our tests? The code looks good, but I'm not sure if this would work for the samples CI test.

Reply:
I'm going to mark this as resolved as I've switched to use `google.auth.default()[1]`